turbopack/crates/turbopack-ecmascript/src/analyzer/imports.rs

```diff
@@ -140,16 +140,56 @@ pub(crate) struct ImportMap {
     /// True if the module is an ESM module due to top-level await.
     has_top_level_await: bool,
 
-    /// Locations of webpackIgnore or turbopackIgnore comments
-    /// This is a webpack feature that allows opting out of static
-    /// imports, which we should respect.
+    /// Locations of [webpack-style "magic comments"][magic] that override import behaviors.
+    ///
+    /// Most commonly, these are `/* webpackIgnore: true */` comments. See [ImportOverrides] for
+    /// full details.
+    ///
+    /// [magic]: https://webpack.js.org/api/module-methods/#magic-comments
+    overrides: HashMap<BytePos, ImportOverrides>,
```

Comment:
I feel `overrides` is a weird name for that.

Maybe `annotations`, `attributes`, `hints` or `magic_comments`